Purchases/SubscriberAttributes/RCSubscriberAttributesManager.m

```diff
@@ -110,7 +111,7 @@ - (void)markAttributesAsSynced:(RCSubscriberAttributeDict)syncedAttributes
 
 - (void)storeAttributeLocallyIfNeededWithKey:(NSString *)key value:(NSString *)value appUserID:(NSString *)appUserID {
     NSString *valueOrEmpty = value ?: @"";
-    NSString * _Nullable currentValue = [self currentValueForAttributeWithKey:key appUserID:appUserID];
+    NSString *_Nullable currentValue = [self currentValueForAttributeWithKey:key appUserID:appUserID];
```

Comment:
can this be nullable instead of _Nullable? If not, I think we normally keep the space between `*` and `_` and AppCode's formatter removes it.

Reply:
shame about AppCode's formatter, but it looks like the setting that adds would add that space would also add a space after any `*`, which would be annoying. Couldn't find a way to separate those